Week1/Gallery/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="ko">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="styles.css" />
+		<title>🐼귀여움이🐱세상을🐶구한다🌏</title>
+	</head>
+
+	<body>
+		<header>🐼귀여움이🐱세상을🐶구한다🌏</header>
```

Comment:
header 는 제목, 로고, 검색 폼, 작성자 이름 등의 요소 를 포함하는 소개 및 탐색에 도움을 주는 콘텐츠를 담는 영역!이기 때문에 정확히는 이 안에서 
```suggestion
      <header><h1>🐼귀여움이🐱세상을🐶구한다🌏</h1></header>
```
이렇게 써주는게 좋답니다! :)